Add info about enabling buildless mode in serverless configuration file (#1789)
kyma-project/serverless: docs/user/00-20-configure-serverless.md

```yaml
   spec:
      enableNetworkPolicies: true
```

## Enable Buildless Mode of Serverless

> You can enable buildless mode of Serverless to skip the image build step for Functions, accelerating prototype development by eliminating the need to build and push custom function images. 

Reviewer: ```suggestion
You can enable buildless mode of Serverless to skip the image build step for Functions, accelerating prototype development by eliminating the need to build and push custom Function images. 
```

> Instead, your code and dependencies are mounted into Kyma-provided runtime images, which shortens the feedback loop, simplifies the architecture, and removes operational barriers, 

Reviewer 2: Dont need to repeat the same content.. Lets just leave the first sentence here

> Instead, your code and dependencies are mounted into Kyma-provided runtime images, which shortens the feedback loop, simplifies the architecture, and removes operational barriers, 

Reviewer: ```suggestion
Instead, your code and dependencies are mounted into Kyma-provided runtime images, which shortens the feedback loop, simplifies the architecture, and removes operational barriers.
```

> allowing developers to focus on business logic rather than containerization and Kubernetes deployment.

Reviewer: ```suggestion
With that, developers can focus on business logic rather than containerization and Kubernetes deployment.
```

   ```yaml
    annotations:
      serverless.kyma-project.io/buildless-mode: "enabled"
   ```

> > [!WARNING]

Reviewer 2: warning should come first, right after the paragraph title

> > Buildless mode is a feature flag and is enabled through annotations. Ensure you understand its implications before enabling it. For more information, see [Buildless Serverless](https://kyma-project.io/#/serverless-manager/user/technical-reference/03-10-buildless-serverless.md).

Reviewer 2: ```suggestion
> Buildless mode is a feature flag and is enabled through annotation. Make sure you read information about [Buildless Serverless](https://kyma-project.io/#/serverless-manager/user/technical-reference/03-10-buildless-serverless.md) before enabling the feature.
```
